distsql/select_result.go

```diff
@@ -290,6 +291,19 @@ func (r *selectResult) updateCopRuntimeStats(ctx context.Context, copStats *copr
 	if r.rootPlanID <= 0 || r.ctx.GetSessionVars().StmtCtx.RuntimeStatsColl == nil || callee == "" {
 		return
 	}
+	if len(r.selectResp.GetExecutionSummaries()) != len(r.copPlanIDs) {
+		logutil.Logger(ctx).Error("invalid cop task execution summaries length",
+			zap.Int("expected", len(r.copPlanIDs)),
+			zap.Int("received", len(r.selectResp.GetExecutionSummaries())))
+		return
+	}
```

Comment:
The log code was already removed, why add it back?

Reply:
I forgot to remove old code. Have removed it.🙈